sql/_08_javasp/answers/case_caution_03.answer

```diff
@@ -0,0 +1,84 @@
+===================================================
+0
+===================================================
+0
+===================================================
+0
+===================================================
+0
+===================================================
+1
+===================================================
+1
+===================================================
+1
+===================================================
+1
+===================================================
+1
+===================================================
+1
+===================================================
+1
+===================================================
+1
+===================================================
+1
+===================================================
+1
+===================================================
+col1    col2    intTest(col1)    
+1     10     1     
+2     10     2     
+3     10     3     
+4     10     4     
+5     10     5     
+6     10     6     
+7     10     7     
+8     10     8     
+9     10     9     
+10     10     10     
+
+===================================================
+col1    col2    intTest(col1)    
+10     10     10     
+9     10     9     
+8     10     8     
+7     10     7     
+6     10     6     
+5     10     5     
+4     10     4     
+3     10     3     
+2     10     2     
+1     10     1     
+
+===================================================
+col1    col2    intTest(99)    
+1     10     99     
+2     10     99     
+3     10     99     
+4     10     99     
+5     10     99     
+6     10     99     
+7     10     99     
+8     10     99     
+9     10     99     
+10     10     99     
+
+===================================================
+col1    col2    intTest(99)    
+10     10     99     
+0     262144     262144     
+9     0     4     
+99     -2096627712     262144     
+262144     262144     426760     
+4     4     4     
+393216     0     327680     
+426760     7     0     
+4     6     null     
+262144     262144     262144     
```

Comment:
Currently, if used 'JAVASP' is in the 'ORDER BY' clause then the query result is a strange
Related http://jira.cubrid.org/browse/CBRD-24687